tests/matrix_free/is_supported_01.output

```diff
@@ -4,20 +4,20 @@ DEAL::FE_BDM<2>(1) supported by MatrixFree: false
 DEAL::FE_Bernstein<2>(1) supported by MatrixFree: true
 DEAL::FE_DGBDM<2>(1) supported by MatrixFree: false
 DEAL::FE_DGNedelec<2>(1) supported by MatrixFree: false
-DEAL::FE_DGP<1,2>(1) supported by MatrixFree: false
+DEAL::FE_DGP<1,2>(1) supported by MatrixFree: true
```

Comment:
Does this has any side effects?

See also the comment by @bergbauer 

> Is this already used in the VectorTools functions etc now due to the changes in is_supported?

Reply:
It should not, because we don't allow for compiling/linking with `DoFHandler<dim, spacedim>` when `spacedim != dim`. So I believe the user gets an error early on. I have a slight hope to be able to tackle #9544 before the release, when this comes more naturally.